ext/nrf/cc310_glue.h

```diff
@@ -13,6 +13,13 @@
 #include <devicetree.h>
 #include <string.h>
 
+/*
+ * Name translation for peripherals with only one type of access available.
+ */
+#ifndef NRF_TRUSTZONE_NONSECURE
```

Comment:
Does this work for `NRF52` because I don't think it will(I could be wrong).
Since `NRF_TRUSTZONE_NONSECURE` is not defiend but NRF_CRYPTOCELL_S is not a register name in `NRF52`

Reply:
This translation only works if NRF_TRUTZONE_NONSECURE is defined, otherwise (nrf52) translation is not needed. See also changes in lines 41 and 46.

Reply:
```suggestion
#ifdef NRF_TRUSTZONE_NONSECURE

Reply:
I get it now, condition updated with CONFIG_ARM_TRUSTZONE_M check